src/modals/confirmation-modal/d-app-operations-confirmation/permission-request-confirmation/permission-request-confirmation.tsx

```diff
@@ -72,6 +74,8 @@ export const PermissionRequestConfirmation: FC<Props> = ({ message }) => {
       publicKey: approver.publicKey
     });
 
+  useNavigationSetOptions({ headerTitle: () => <HeaderTitle title={'Confirm Connection'} /> }, []);
```

Comment:
according to style guide if you are using hardcoded string as prop do not use curly braces
`title="Confirm Connection"`